frontends/web/src/locales/en/app.json

```diff
@@ -984,9 +984,9 @@
     "unlocked3": "Tap the touch button when the LED lights up"
```

Comment:
Above:
- Get started
- Open in external block explorer
- Pending transaction
- Upgrade firmware (twice)
- Do you want to upgrade the firmware…